src/tigerbeetle/cli.zig

```diff
@@ -55,6 +55,11 @@ const usage = fmt.comptimePrint(
     \\        will be used.
     \\        "addresses[i]" corresponds to replica "i".
     \\
+    \\  --standby-count=<integer>
```

Comment:
Good question: I think the behavior is the same as when you start replicas with different lengths of addresses: replicas disagree about the configuration, and so the behavior is undefined. 

:thinking: I wonder if `epoch` should be a hash of configuration, rather than a counter? That seems like a conceptually nice way to solve this kind of problem. Though, I _think_ we might need ordering on the epochs, so _just_ a hash won't work.